spec/operations/classrooms/student_index_spec.rb

```diff
@@ -6,18 +6,12 @@
   let(:classroom) { Classroom.create! join_token: 'abc', zooniverse_group_id: "asdf" }
 
   it 'returns classrooms the current user is a student of' do
-    allow(client).to receive_message_chain(:panoptes, :paginate).and_return({"user_groups" => [{"id" => classroom.zooniverse_group_id}]})
-    allow(client).to receive(:join_user_group).and_return(true)
```

Comment:
This and line 18 below (`(:panoptes, :post)`) are still necessary for the double.